### PR TITLE
Support for API 600/800/1000/1200 for ServerCertificate

### PR DESCRIPTION
**Pull request by chebroluharika**
**Description**
Support for API 600/800/1000/1200 for ServerCertificate

Issues Resolved
NA

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/server_certificate.pp [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# Retrieves the web server certificate
+oneview_server_certificate{'Get Certificates from RemoteIP':
+    ensure  => 'get_certificate',
+    require => Oneview_server_certificate['Import Certificates from RemoteIP'],
+    data    => {
+      remoteIp  => $remoteip
+    }
+}
+
+oneview_server_certificate{'sc1 Retrieve Certificates':
```
Comment (soodpr) [inline examples/server_certificate.pp]: i think its better to rename the resource title as "Retrieve Server Certificates"

File: spec/shared_context.rb [hunk 2]
```diff
@@ -39,6 +39,14 @@
   end
 end
 
+RSpec.shared_context 'shared context Oneview API 600', a: :b do
+  before :each do
+    api_version = 600
```
Comment (soodpr) [inline spec/shared_context.rb:44]: not sure if you really need shared context if you are declaring API version in the unit test cases

**Review by soodpr — CHANGES_REQUESTED** [review]
Endpoint-support.md file is missing

**Review by soodpr — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,9 @@
-# v2.4.1 (unreleased)
+# v2.5.0 (unreleased
```
Comment (jyothisgm) [inline CHANGELOG.md]: close the bracket

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,9 @@
...
+- This release supports OneView Rest API versions 800/1000/1200 minimally for Server Certificate.
```
Comment (jyothisgm) [inline CHANGELOG.md]: You dont need to specify the resource here also just say for the resources listed below.

File: endpoints-support.md [hunk 5]
```diff
@@ -153,6 +153,12 @@
 |<sub>/rest/sas-logical-interconnect-groups/{id}</sub>                                    | GET      | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/sas-logical-interconnect-groups/{id}</sub>                                    | PUT      | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/sas-logical-interconnect-groups/{id}</sub>                                    | DELETE   | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|     **Server Certificate**                                                                                                   |
```
Comment (jyothisgm) [inline endpoints-support.md:162]: If you are supporting for 600 shouldn't we note that in changelog as well

**Review by jyothisgm — CHANGES_REQUESTED** [review]

File: examples/server_certificate.pp (hunk 1 above)
Comment (chebroluharika) [reply examples/server_certificate.pp]: done

**Review by jyothisgm — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 3 above)
Comment (chebroluharika) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 4 above)
Comment (chebroluharika) [reply CHANGELOG.md]: done

File: endpoints-support.md (hunk 5 above)
Comment (chebroluharika) [reply endpoints-support.md:162]: specified in CHANGELOG

**Pull request merged**